### PR TITLE
Debugger 2.9.1.14 Fix disassembly when in middle of data

### PR DESCRIPTION
**Pull request by Michaelangel007**
Small fix.

### CODE REVIEW

**Comment by Michaelangel007** [comment]
Repro:
```
ASC 7D0:7D7
ASC 7D8:7DF
7CF:EA 40 41 42 43 44 45 46 47 48
U 7D7
```

Before: 
2 9 1 14_before

After:
2 9 1 14_after

**Comment by Michaelangel007** [comment]
@tomcw When you have a quick moment to double check & test.

Data markup & disassembly still has other bugs (i.e. DB2 <addr> doesn't tag addr+1 as data, etc.) but this was a low-hanging fruit one.

File: source/Debugger/Debugger_Assembler.cpp [hunk 1]
```diff
@@ -498,7 +498,10 @@ int  _6502_GetOpmodeOpbyte ( const int nBaseAddress, int & iOpmode_, int & nOpby
 		if ( pData_ )
 			*pData_ = pData;
 
-		nSlack = pData->nEndAddress - pData->nStartAddress + 1; // *inclusive* KEEP IN SYNC: _CmdDefineByteRange() CmdDisasmDataList() _6502_GetOpmodeOpbyte() FormatNopcodeBytes()
+		const DWORD nStartAddress = pData->nStartAddress;
```
Comment (tomcw) [inline source/Debugger/Debugger_Assembler.cpp:501]: You don't seem to be using this `nStartAddress` variable. Delete this line?

Comment (Michaelangel007) [reply source/Debugger/Debugger_Assembler.cpp]: Thanks for the catch! Fixed.

**Comment by tomcw** [comment]
AFAICT, this looks fine - so merge it in.

**Pull request merged**